src/TestFramework/Coverage/CodeCoverageData.php

```diff
@@ -98,73 +97,44 @@ static function ($testMethods) {
         return \count($coveredLineTestMethods) > 0;
     }
 
-    public function hasTestsOnLine(string $filePath, int $line): bool
```

Comment:
As a side note, thanks to running infection on itself i discovered that these two methods weren't useful, as any mutation that made them return `true` was never caught, and i couldn't produce tests to catch them.